src/directives.cpp

```diff
@@ -3,15 +3,19 @@
 namespace YAML {
 Directives::Directives() : version{true, 1, 2}, tags{} {}
 
-const std::string Directives::TranslateTagHandle(
+std::string Directives::TranslateTagHandle(
     const std::string& handle) const {
+  std::string result;
   auto it = tags.find(handle);
   if (it == tags.end()) {
     if (handle == "!!")
-      return "tag:yaml.org,2002:";
```

Comment:
Removing const makes sense, but why do you need to get rid of the early returns?

Reply:
@jbeder I was thinking that RVO wouldn't work because several return statements can influence on it. Ok, I have checked it on clang compiler, everything is ok. I have reverted this part of changes and will research it on different compilers. It was wrong to add this changes to this PR.